### ISSUE TITLE
Support `--var` flag in apply-manifest

### ISSUE THREAD

**Issue by andy-paine** [cloudfoundry/cli#1793]
**What's the user value of this feature request?**
Users who are using the `apply-manifest` workflow can leverage the variable interpolation syntax that `cf7 push` uses.

**Who is the functionality for?**
Anyone wanting to DRY their manifests, including CI systems.

**How often will this functionality be used by the user?**
Every time the manifest changes.

**Who else is affected by the change?**
Should be a new feature so won't affect people.

**Is your feature request related to a problem? Please describe.**
It is fairly common to want to template your manifest so you can change routes, env vars, app name etc. between environments. `cf7 push` allows you to do this by adding `((var-name))` templates to the manifest and `--var` flags to `cf7 push`. It would be nice for the `apply-manifest` workflow to support this templating too as otherwise we need to resort to external templating (Jinja, ERB, envsubst etc.)

**Describe the solution you'd like**
The same templating syntax and command line flags as `cf7 push`

**Describe alternatives you've considered**
Not doing this and making our CI systems have to include an external templating engine :disappointed:

**Comment by abbyachau** [comment]
Hey @andy-paine thanks for this feature request. If you are up for it, we would love a pull request to support this feature. If so, lmk and I can help write up a workflow template to help with this effort. Thanks!

**Comment by reid47** [comment]
We recently refactored the parts of `cf7 push` that deal with parsing the manifest, and it led to us rewriting the manifest parser package. The new package (called `pushmanifestparser`) is only used in `push`, but the old one (called `manifestparser`) is still used in `apply-manifest`.

We made a chore to update `apply-manifest` to use the new one (and probably rename it to something less `push`-specific). Here's that chore: https://www.pivotaltracker.com/story/show/168185546

But this feature could be a good way to drive the completion of that work.
